Update connect-sensors.md (#90380)
Azure/azure-content: articles/defender-for-iot/organizations/connect-sensors.md

> [!IMPORTANT]
> Some organizations must define firewall rules by IP addresses. If this is true for your organization, it's important to know that the Azure public IP ranges are updated weekly.
>

> > Make sure to download the new JSON file each week and make the required changes on your site to correctly identify services running in Azure. You'll need the updated IP ranges for **AzureIoTHub**, **Storage**, and **EventHub**. The latest IP ranges can be [found here.](https://www.microsoft.com/en-us/download/details.aspx?id=56519)

Reviewer: ```suggestion
> Make sure to download the new JSON file each week and make the required changes on your site to correctly identify services running in Azure. You'll need the updated IP ranges for **AzureIoTHub**, **Storage**, and **EventHub**. See the [latest IP ranges](https://www.microsoft.com/en-us/download/details.aspx?id=56519).
```

    - **Threat Intelligence**: `*.blob.core.windows.net`
    - **Eventhub**: `*.servicebus.windows.net`

> 1. Azure public IP addresses are updated weekly. If you must define firewall rules based on IP addresses, make sure to download the new JSON file each week and make the required changes on your site to correctly identify services running in Azure. You'll need the updated IP ranges for **AzureIoTHub**, **Storage**, and **EventHub**. These IP ranges can be [Downloaded here.](https://www.microsoft.com/en-us/download/details.aspx?id=56519)

Reviewer: ```suggestion
1. Azure public IP addresses are updated weekly. If you must define firewall rules based on IP addresses, make sure to download the new JSON file each week and make the required changes on your site to correctly identify services running in Azure. You'll need the updated IP ranges for **AzureIoTHub**, **Storage**, and **EventHub**. See the [latest IP ranges](https://www.microsoft.com/en-us/download/details.aspx?id=56519).
```
